drivers/pps/sysfs.c

```diff
@@ -97,3 +98,28 @@ const struct attribute_group *pps_groups[] = {
 	&pps_group,
 	NULL,
 };
+
+/*
+ * Module parameters
+ */
+
+int pps_max_sources = 32;
```

Comment:
This should probably default to the previous limit of 16; we probably shouldn't change the defaults out from other users.